torch/functional.py

```diff
@@ -825,3 +826,10 @@ def gels(input, A, out=None):
     warnings.warn("torch.gels is deprecated in favour of torch.lstsq and will be removed in "
                   "the next release. Please use torch.lstsq instead.", stacklevel=2)
     return torch.lstsq(input, A, out=out)
+
+
+def align_tensors(*tensors):
+    if not torch._C._BUILD_NAMEDTENSOR:
+        raise RuntimeError('NYI: torch.align_tensors is experimental and a part '
+                           'of our named tensors project.')
```

Comment:
NYI == 'not yet implemented,' but this is implemented right? I'd think the error should just tell them they need to have built with the experimental build option to use this

Reply:
I don't want any users to start using this as an experimental feature right now so I wanted to tell them "hey, nothing to see here yet, move along".